src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/BrokeredServices/ServiceBrokerFactory.cs

```diff
@@ -70,11 +78,22 @@ public async Task CreateAsync()
 
         _container = await BrokeredServiceContainer.CreateAsync(_exportProvider, _cancellationTokenSource.Token);
 
+        IServiceBroker serviceBroker;
+        try
```

Comment:
I don't think you need a try/catch around this.  It shouldn't fail.  

The try catch is around the `OnServiceBrokerInitialized` so we don't throw if an extension throws initializing.